src/utilities/templates/flatpages/flatpage-form.html

```diff
@@ -0,0 +1,33 @@
+{% extends "flatpages/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block head %}
+    {{ form.media }}
+{% endblock %}
+
+{% block head_title%}
+
+    {% if not update%}
+        Create Page | 
+    {% else %}
+        Update Page | 
+    {% endif %}
+
+{% endblock%}
+
+{% block content %}
+    <div>
+        <form action="" method="post">
+            {% csrf_token %}
+
+            {{ form|crispy }}
+
+            <a href="{% url 'utilities:flatpage-list' %}" role="button" class="btn btn-danger">Cancel</a>
+            <input type="submit" value="Submit" class="btn btn-success"/>
+            {% if update %}
+                <a href="{% url 'utilities:flatpage-delete' pk=flatpage.pk %}" role="button" class="btn btn-danger">Delete</a>
```

Comment:
Don't have both Cancel and Delete?  The button combo's here don't mathc the rest of the website, take a look at what the other CRUD views do, especially the Announcements (cus it's simple) and copy that.

Reply:
Removed cancel button from form.html